doc/adr/release.md

```diff
@@ -0,0 +1,1019 @@
+system ensures that the appropriate works are distributed according to the users
+use of the [Controls](#controls) and in a way that associates their works to an
+[Release Identifier](#release-identifier) specified by the user.
+
+### Concrete solution
+
+An executable named `create_release`.
+
+#### Parameters
+
+##### `RELEASE_ID` (Param)
+
+- Position: 0
+- Type: String
+- Required
+
+The [Release Identifier](#release-identifier).
+
+#### Release Configuration
+
+The [Release Configuration](#release-configuration) acts that the main set of
+[Controls](#controls) provided to the user. As [Controls](#controls), the values
+user provides for the [Release Configuration](#release-configuration) determines
+which [Objectives](#objective-notion) are [User Objectives](#user-objective),
+what parameters are used in those [Objectives](#objective-notion) and ultimately
+what the contents of the [Release Plan](#release-plan).
+
+For the `create_release` executable to work their needs to be a `release.yml` or
+`release.yaml` file at the root of the git repository using IOGX.
+
+The release configuation file will have the fallowing options...
+
+##### `title.prefix`
+
+- Type: String
+- Default: Empty String
+
+A prefix combined with the [Release Identifier](#release-identifier)
+to form a release title to be used in content where human readability is important.
+
+##### `description.text`
+
+- Type: String
+- Default: Empty String
+
+A description of the release.
+
+##### `description.include-github-generated-release-notes`
+
+- Type: Bool
+- Default: False
+
+Whether to include the release notes generated by GitHub in the descriptions of
+the release.
+
+##### `git.tag.always-publish`
+
+- Type: Bool
+- Default: False
+
+If the git tag should be published even if no release objectives call for it.
+
+##### `git.tag.prefix`
+
+- Type: String
+- Default: Empty String
+
+A string prepended to the [Release Identifier](#release-identifier) to
+form the git tag representing the release in git repositories.
+
+##### `github.release.always-publish`
+
+- Type: Bool
+- Default: False
+
+If the GitHub release should be published even if no other release objectives
+call for it.
+
+##### `github.release.assets`
+
+- Type: Mapping (String -> `flake-output-path`)
+- Default: Empty Map
+
+A mapping from the desired name of the asset in the GitHub release to the file
+to exist under that asset name.
+
+> Note: All values (`flake-output-path`s) must resolve to files.
+
+<!-- -->
+> Note: There cannot be any naming conflicts between names to be used as asset
+names in the GitHub release.
+
+Example
+
+The configuration...
+
+```yaml
+github:
+  release:
+    assets:
+      foo-x86_64-linux:
+        output: packages.x86_64-linux.foo
+        path: /lib/foo.so
+      foo2-x86_64-linux:
+        output: packages.x86_64-linux.foo
+        path: /lib/foo2.so
+      bar:
+        output: packages.x86_64-linux.bar
+      biz-x86_64-linux: biz
```

Comment:
Just to be confirm:  `biz-x86_64-linux: biz` -> 
This will look into the flake output `biz`, which supposedly is a system-independent file.
So I would expect something like this instead:
```yaml
biz : biz
```

Reply:
Yeah, maybe the example giving it that name on GitHub makes things a little confusing.

Reply:
Done 786989bf51c4683f0e4153598fad5be8bd2c6f55